src/audio_player.rs

```diff
@@ -0,0 +1,90 @@
+use crate::sync_wrapped;
+use bevy::prelude::{AudioSinkPlayback, Component, Vec3};
+
+#[derive(Component)]
+pub struct AudioListener {
+    pub previous_position: Vec3,
+}
+
+impl Default for AudioListener {
+    fn default() -> Self {
+        AudioListener {
+            previous_position: Vec3::default(),
+        }
+    }
+}
+
+#[derive(Component)]
+pub struct AudioSource {
+    pub name: &'static str,
+}
+
+#[derive(Component)]
+pub struct AudioSourcePlayer {
+    pub name: &'static str,
+    pub fmod_event: sync_wrapped::EventInstance,
+    pub previous_position: Vec3,
```

Comment:
Could be a `Bundle` with `Transform` again.

Reply:
By the way, if we bundle it with Transform then with the current implementation the Attributes3d will be calculated every frame even for 2D sounds and menu music. Either way we should probably add an example for 2D audio soon so it will also be clear for ourselves what that looks like